classes/Cart.php

```diff
@@ -1629,7 +1638,12 @@ public function updateQty(
 
             // Quantity for product pack
             if (Pack::isPack($id_product)) {
-                $result2['quantity'] = Pack::getQuantity($id_product, $id_product_attribute, null, $this);
+                $result2['quantity'] = Pack::getQuantity(
+                    $id_product,
+                    $id_product_attribute,
+                    null,
+                    $this instanceof Cart ? $this : null
```

Comment:
Looks a bit strange. How could this be false? How `$this` could not be an instance of the class in which file it belongs?

Reply:
Explanation here: https://github.com/PrestaShop/PrestaShop/pull/26913/files#r772394422